apps/amiapp_flutter/lib/src/data/config.dart

```diff
@@ -1,58 +1,87 @@
+import 'package:customer_io/config/in_app_config.dart';
+import 'package:customer_io/config/push_config.dart';
+import 'package:customer_io/customer_io_enums.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 class CustomerIOSDKConfig {
-  String siteId;
-  String apiKey;
-  String? trackingUrl;
-  double? backgroundQueueSecondsDelay;
-  int? backgroundQueueMinNumOfTasks;
-  bool screenTrackingEnabled;
-  bool deviceAttributesTrackingEnabled;
-  bool debugModeEnabled;
+  final String cdpApiKey;
+  final String? migrationSiteId;
+  final Region? region;
+  final bool? debugModeEnabled;
+  final bool? screenTrackingEnabled;
+  final bool? autoTrackDeviceAttributes;
+  final String? apiHost;
+  final String? cdnHost;
+  final int? flushAt;
+  final int? flushInterval;
+  final InAppConfig? inAppConfig;
+  final PushConfig pushConfig;
```

Comment:
I was expecting these changes to be part of sample app updates ticket that I'll be working on soon. Since we do need these changes, I'm okay with merging this now and finalizing it along with other sample app changes.

Reply:
Yeah, i made them first to make your life easier :D second, because I wanted to ensure sample app is working with new source 😅